src/components/Forms/CopyTraderForm/CopyTraderForm.js

```diff
@@ -257,6 +258,14 @@ const CopyTraderForm = ({ provider, onClose, onSuccess }) => {
     handleSubmit(onSubmit);
   };
 
+  const redirectToHelp = () => {
+    if (typeof window !== "undefined") {
```

Comment:
This is not needed, gatsby static site rendering won't run this method.

Reply:
you sure, because previously it broke the build?

Reply:
can we merge this now and I will clean later, because i have to change in test as well

Reply:
Only when the code is run at init time. Not inside the functions.